android/build.gradle

```diff
@@ -220,7 +220,7 @@ android {
   splits.abi {
     enable true
     reset()
-    include 'x86', 'armeabi-v7a'
+    include 'armeabi-v7a'
```

Comment:
For the same reason. It's temporary. First I wanted to update NDK at least for arm architecture to get it (https://developer.android.com/studio/projects/add-native-code.html ) work